egs/wsj/s5/utils/combine_ali_dirs.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+# Copyright 2016  Xiaohui Zhang  Apache 2.0.
+
+# This script operates on alignment directories, such as exp/tri4a_ali
+
+# Begin configuration section. 
+cmd=run.pl
+extra_files= # specify addtional files in 'src-data-dir' to merge, ex. "file1 file2 ..."
```

Comment:
Is this extra_files required? It does not seem to be used, and also comment is incorrect.